### ISSUE TITLE
'TaskQueueManager' object has no attribute 'callback_obj'

### ISSUE THREAD

**Issue by julien-lang** [ansible/ansible#40453]
##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION

```
2.5.3
```

##### OS / ENVIRONMENT

Debian GNU/Linux 9.4 (stretch)
Ansible 2.5.3-1ppa~xenial

##### SUMMARY

When I run my usual playbook with ansible 2.5.3, I got the following error message and ansible stop

` 'TaskQueueManager' object has no attribute 'callback_obj' `
This issue happened after I updated my system and got an Ansible update. If I restore the previous version (2.4.2), I have no error.

##### STEPS TO REPRODUCE

I don't have a clear steps to reproduce the issue yet.

##### EXPECTED RESULTS

The playbook runs

##### ACTUAL RESULTS

```
ERROR! Unexpected Exception, this is probably a bug: 'TaskQueueManager' object has no attribute 'callback_obj'
to see the full traceback, use -vvv
```

I did not have a chance to run the same command in "-vvv". The issue happens on a production server and I downgraded as soon as I can. But I'll try to reproduce it on a different system and run -vvv

**Comment by julien-lang** [comment]
Looking at the installed code on ansible 2.5.3 on my Debian system (/usr/lib/python2.7/dist-packages/ansible), I might have found something suspicious.

In the _executor/task_queue_manager.py_ file, there is this code:
```python
            callback_obj = callback_plugin()
            try:
                callback_obj.set_options()
            except AttributeError:
                    display.deprecated("%s callback, does not support setting 'options', it will work for now, "
                                       " but this will be required in the future and should be updated, "
                                       " see the 2.4 porting guide for details." % self.callback_obj._load_name, version="2.9")
            self._callback_plugins.append(callback_obj)
```

The use of **self.callback_obj** does not seem right since the object define this variable. But in this function, there is a local variable named _callback_obj_ define. And nowhere else in the code there is a such variable name (grep -R '.callback_obj').

So I suspect the fix to be:
```diff
-                                       " see the 2.4 porting guide for details." % self.callback_obj._load_name, version="2.9")
+                                       " see the 2.4 porting guide for details." % callback_obj._load_name, version="2.9")
```

**Comment by yodatak** [comment]
I got the same on ansible 2.5.3 , it wiil be fix on ansible 2.5.4 right ?

**Comment by greenscar** [comment]
In case it helps anyone:

With regards to awx and this problem, I removed the task docker container & removed the image from my local repo. I then reran a `docker-compose -f _______ up -d`  (yeah we have a customized docker setup). The point here: By removing the docker image & index, it reinstalled ansible & thus provided me the later version in my task container.
